client/my-sites/themes/sheet.jsx

```diff
@@ -51,6 +53,14 @@ const ThemeSheet = React.createClass( {
 		this.props.dispatch( signup( this.props ) );
 	},
 
+	onDownloadButtonClick() {
+		if ( this.props.download ) {
```

Comment:
Does this mean that if it's empty we assume it's on .org? Is it a fair assumption to make?